### DISCUSSION TITLE
Profiles Sort Order

### DISCUSSION THREAD

**Discussion by cburkins** [Eugeny/tabby#5665] [Q&A]
Big fan of Tabby, and big fan of saved "Profiles & connections".   I'm organizing my profiles into Groups.  I can't quite figure out the sort order of Profiles within each group.  It seems sort of random.   I would prefer alphabetical if possible.   Some of my hosts are just numerical sequences (e.g. kermit01, kermit02), so alphabetical would be nice.

-Chad

**Comment by cburkins** [comment]
By chance, does anyone have tips on how to set the sort order of the saved profiles in each group ?

**Comment by nimdasx** [comment]
i need this too (sort in profiles and connection), 
btw thanks for this great app

**Comment by tarocjsu** [comment]
Look like no one care about this topic or question ?

**Comment by nimdasx** [comment]
really need this, thanks Eugeny

**Comment by tarocjsu** [comment]
Oooops....still no one can answer....

**Comment by Clem-Fern** [comment]
Hi guys, I have opened the PR #8726 which fix this issue. Profiles are alphanumerically sorted before being displayed:

![image](https://github.com/Eugeny/tabby/assets/20025949/b9b28f54-871e-4d03-92c4-6e4bb6350878)

**Comment by Clem-Fern** [comment]
Fixed in v1.0.198
